core/server/data/migrations/versions/5.5/2022-07-18-14-29-add-comment-reporting-permissions.js

```diff
@@ -0,0 +1,12 @@
+const {combineTransactionalMigrations, addPermissionWithRoles} = require('../../utils');
+
+module.exports = combineTransactionalMigrations(
```

Comment:
No need for the use of combine for a single migration

Reply:
Fixed! 👌